src/common/casualReferences.ts

```diff
@@ -44,6 +53,15 @@ export function tomorrow(reference: ReferenceWithTimezone): ParsingComponents {
     return component;
 }
 
+export function theDayAfterTomorrow(reference: ReferenceWithTimezone): ParsingComponents {
```

Comment:
Similarly, I would make it `theDayAfter(reference: ReferenceWithTimezone, nDays: number)`